integration_tests/tests/vault/slash.rs

```diff
@@ -120,7 +120,7 @@ mod tests {
             .do_slash(
                 &vault_root,
                 &ncn_root.ncn_pubkey,
-                &slasher,
+                slasher,
```

Comment:
was this just broken?

Reply:
It was working before, just modified from running cargo clippy